### PR TITLE
Alex LaFroscia, Deliverable 4

### PR DESCRIPTION
**Pull request by alexlafroscia**

### CODE REVIEW

File: com/laboon/WorldTest.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      char c = str.charAt(i);
+      if (c == 'X' || c == 'x') {
+        numLivingInString++;
+      }
+    }
+
+    assertEquals(numLivingInWorld, numLivingInString);
+  }
+}
+
```
Comment (laboon) [inline com/laboon/WorldTest.java:69]: Should add some checks for edge cases, specific values (i.e. specific board outputs).  but otherwise good

**Comment by alexlafroscia** [comment]
Mind if I close this?

**Comment by laboon** [comment]
No, go ahead.  I was going to just close them all at the end of the semester in case people wanted to look/review before then.

**Comment by alexlafroscia** [comment]
Oh okay, gotcha.

**Pull request closed without merging**